src/test/java/nl/tudelft/ewi/devhub/webtests/views/PullRequestView.java

```diff
@@ -0,0 +1,17 @@
+package nl.tudelft.ewi.devhub.webtests.views;
+
+import org.openqa.selenium.WebDriver;
+
+public class PullRequestView extends AuthenticatedView {
+
+	public PullRequestView(WebDriver driver) {
+		super(driver);
+	}
+
+	@Override
+	protected void invariant() {
+		// TODO Auto-generated method stub
```

Comment:
Maybe we should introduce an abstraction between the pull request activity feed view, and the pull request diff view, that checks the breadcrumb and header (just as `ProjectSidebarView` does for pages with a sidebar)